draft-ietf-webpush-encryption.md

```diff
@@ -416,7 +421,7 @@ Shared ECDH secret (ecdh_secret):
 
 : kyrL1jIIOHEzg3sM2ZWRHDRB62YACZhhSlknJ672kSs
 
-Pseudo-random key for key combining (PRK_key):
+Pseudorandom key (PRK) for key combining (PRK_key):
```

Comment:
That's true, but this is the first time we have prose on which this can be attached.  The uses earlier in the document are just symbols in "code".